scalding-commons/src/main/java/com/twitter/scalding/commons/scheme/KeyValueByteScheme.java

```diff
@@ -24,6 +26,11 @@ public KeyValueByteScheme(Fields fields) {
     super(fields, BytesWritable.class, BytesWritable.class);
   }
 
+  @Override
+  public void sourceConfInit(FlowProcess<JobConf> flowProcess, Tap<JobConf, RecordReader, OutputCollector> tap, JobConf conf) {
+    conf.setInputFormat(CombineSequenceFileInputFormat.class);
```

Comment:
EB is opt-in, you have to set elephantbird.use.combine.input.format=true 
How do you measure "better" for hadoop's impl. The code change is simpler in the EB case.

You can make input format configurable.